src/components/projects-summary/projects-summary.html

```diff
@@ -30,7 +30,7 @@ <h4 class="catalog-modal-title">
     </a>
   </div>
   <div ng-if="$ctrl.projects && $ctrl.projects.length" class="catalog-project-summary-list">
-    <div ng-if="$ctrl.totalProjects > $ctrl.maxDisplayProjects" class="projects-count">
+    <div class="projects-count">
       <strong>{{$ctrl.maxDisplayProjects}}</strong>
       of
       <strong>{{$ctrl.totalProjects}}</strong>
```

Comment:
@spadgett @cdcabrera Yes the project count should be displayed as well. As long as there is at least 1 project, the project count and view all link should be shown.